docs/source/configuration/authn-jwt.mdx

```diff
@@ -37,7 +29,7 @@ These are the high-level steps of JWT-based authentication with the Apollo Route
 
 ## Turning it on
 
-You enable JWT authentication for your router with the following steps:
+If you issue JWTs via a popular third-party IdP (Auth0, Okta, PingOne, etc.), enabling JWT authentication in your router is a two step process described below. If you use your own custom IdP, [advanced configuration is required](#creating-your-own-jwks-advanced).
```

Comment:
```suggestion
If you use your own custom IdP, [advanced configuration is required](#creating-your-own-jwks-advanced).

Otherwise, if you issue JWTs via a popular third-party IdP (Auth0, Okta, PingOne, etc.), enabling JWT authentication in your router is a two step process described below. 
```